sympy/solvers/ode/tests/test_systems.py

```diff
@@ -781,11 +951,157 @@ def test_linear_3eq_order1_type4_slow():
 
 
+def test_linear_neq_order1_type2_big_test_cases():
+    i, r1, c1, r2, c2, t = symbols('i, r1, c1, r2, c2, t')
+    x1 = Function('x1')
+    x2 = Function('x2')
+    eq1 = r1*c1*Derivative(x1(t), t) + x1(t) - x2(t) - r1*i
+    eq2 = r2*c1*Derivative(x1(t), t) + r2*c2*Derivative(x2(t), t) + x2(t) - r2*i
+    eq = [eq1, eq2]
+    _x1 = c1 ** 2 * r1 ** 2 + 2 * c1 ** 2 * r1 * r2 + c1 ** 2 * r2 ** 2 - 2 * c1 * c2 * r1 * r2 + 2 * c1 * c2 * r2 ** 2 + c2 ** 2 * r2 ** 2
+    _x2 = Integral(i * r2 * exp(-sqrt(_x1) * t / (2 * c1 * c2 * r1 * r2) + t / (2 * c2 * r2)
+                                + t / (2 * c2 * r1) + t / (2 * c1 * r1)) / sqrt(_x1), t)
+    _x3 = Integral(-i * r2 * exp(sqrt(_x1) * t / (2 * c1 * c2 * r1 * r2) + t / (2 * c2 * r2)
+                                 + t / (2 * c2 * r1) + t / (2 * c1 * r1)) / sqrt(_x1), t)
+    _x4 = exp(sqrt(_x1) * t / (2 * c1 * c2 * r1 * r2) - t / (2 * c2 * r2) - t / (2 * c2 * r1) - t / (2 * c1 * r1))
+    _x5 = exp(-sqrt(_x1) * t / (2 * c1 * c2 * r1 * r2) - t / (2 * c2 * r2) - t / (2 * c2 * r1) - t / (2 * c1 * r1))
+    sol = [
+        Eq(x1(t),
+           - 2 * C1 * _x5 * c2 * r2 / (sqrt(_x1) + c1 * r1 + c1 * r2 - c2 * r2)
+           - 2 * C2 * _x4 * c2 * r2 / (-sqrt(_x1) + c1 * r1 + c1 * r2 - c2 * r2)
+           - 2 * _x2 * _x4 * c2 * r2 / (-sqrt(_x1) + c1 * r1 + c1 * r2 - c2 * r2)
+           - 2 * _x3 * _x5 * c2 * r2 / (sqrt(_x1) + c1 * r1 + c1 * r2 - c2 * r2)),
+        Eq(x2(t), C1 * _x5 + C2 * _x4 + _x2 * _x4 + _x3 * _x5),
+    ]
+    assert dsolve(eq) == sol
+    assert checksysodesol(eq, sol) == (True, [0, 0])
+
+
+def _de_lorentz_solution():
+    m = Symbol("m", real=True)
+    q = Symbol("q", real=True)
+    t = Symbol("t", real=True)
+
+    e1, e2, e3 = symbols("e1:4", real=True)
+    b1, b2, b3 = symbols("b1:4", real=True)
+    v1, v2, v3 = symbols("v1:4", cls=Function, real=True)
+
+    eqs = [
+        -e1 * q + m * Derivative(v1(t), t) - q * (-b2 * v3(t) + b3 * v2(t)),
+        -e2 * q + m * Derivative(v2(t), t) - q * (b1 * v3(t) - b3 * v1(t)),
+        -e3 * q + m * Derivative(v3(t), t) - q * (-b1 * v2(t) + b2 * v1(t))
+    ]
+
+    x1 = sqrt(b1 ** 2 + b2 ** 2 + b3 ** 2)
+    x2 = exp(q * t * sqrt(-b1 ** 2 - b2 ** 2 - b3 ** 2) / m)
+    x3 = 1 / (b1 ** 2 * x1 * x2 + b2 ** 2 * x1 * x2)
+    x4 = 1 / (2 * b1 ** 2 * m * x2 + 2 * b2 ** 2 * m * x2 + 2 * b3 ** 2 * m * x2)
+    x5 = Integral(
+        b1 * b3 * e1 * q / (b1 ** 2 * m + b2 ** 2 * m + b3 ** 2 * m)
+        + b2 * b3 * e2 * q / (b1 ** 2 * m + b2 ** 2 * m + b3 ** 2 * m)
+        + b3 ** 2 * e3 * q / (b1 ** 2 * m + b2 ** 2 * m + b3 ** 2 * m), t)
+    x6 = 1 / (2 * b1 ** 3 * b3 * m + 2 * I * b1 ** 2 * b2 * m * x1 + 2 * b1 * b2 ** 2 * b3 * m
+              + 2 * b1 * b3 ** 3 * m + 2 * I * b2 ** 3 * m * x1 + 2 * I * b2 * b3 ** 2 * m * x1)
+    x7 = Integral(
+        b1 ** 2 * e3 * q * x4 - b1 * b3 * e1 * q * x4 + I * b1 * e2 * q * x1 * x4
+        + b2 ** 2 * e3 * q * x4 - b2 * b3 * e2 * q * x4 - I * b2 * e1 * q * x1 * x4, t)
```

Comment:
I don't understand why you've added all of these spaces. Is that something that your editor does automatically? It's easier to read with the spacing from str and careful line breaks:
```python
x7 = Integral(b1**2*e3*q*x4 - b1*b3*e1*q*x4 + I*b1*e2*q*x1*x4
            + b2**2*e3*q*x4 - b2*b3*e2*q*x4 - I*b2*e1*q*x1*x4, t)
```

Reply:
Most probably it's the problem with my terminal, I would get it fixed.